### PR TITLE
Add contracts controller tests

### PR DESCRIPTION
**Pull request by hectorgomezv**
This PR adds some tests for `ContractsController`. It covers the usual scenarios:
- A contract is successfully retrieved.
- Config Service fails.
- Transaction Service fails.
- Transaction Service returns a malformed item (validation error).

### CODE REVIEW

File: src/routes/contracts/contracts.controller.spec.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  mockNetworkService,
+  TestNetworkModule,
+} from '../../datasources/network/__tests__/test.network.module';
+import { DomainModule } from '../../domain.module';
+import { chainBuilder } from '../../domain/chains/entities/__tests__/chain.builder';
+import { contractBuilder } from '../../domain/contracts/entities/__tests__/contract.builder';
+import { ValidationModule } from '../../validation/validation.module';
+import { ContractsModule } from './contracts.module';
+
+describe('Contracts controller', () => {
```
Comment (fmrsabino) [inline src/routes/contracts/contracts.controller.spec.ts:24]: Missing closing the app after the tests (`afterAll`)

**Review by fmrsabino — CHANGES_REQUESTED** [review]
LGTM! One small comment.

File: src/routes/contracts/contracts.controller.spec.ts (hunk 1 above)
Comment (hectorgomezv) [reply src/routes/contracts/contracts.controller.spec.ts:24]: True 🙈, good catch, fixed in [39b407d](https://github.com/safe-global/safe-client-gateway-nest/pull/304/commits/39b407d1bfcdd00e543e0b867aff491f0cae3601)

**Review by fmrsabino — APPROVED** [review]
🚀

**Pull request merged**